bin/merge

```diff
@@ -37,26 +45,43 @@ if [[ "$merged" == "false" ]]; then
     cd $directory
     git checkout -b $tmp_branch $base_ref
     git pull --ff-only $pr_clone $pr_ref
+
     old_version=($(sed 's/.*"\(.*\)"/\1/' version.sbt | tr "." "\n"))
-    if [[ -e "CHANGELOG/major.md" ]]; then
-        new_version="$((old_version[0] + 1)).0.0"
-        git mv CHANGELOG/major.md CHANGELOG/${new_version}.md
-    elif [[ -e "CHANGELOG/minor.md" ]]; then
-        new_version="${old_version[0]}.$((old_version[1] + 1)).0"
-        git mv CHANGELOG/minor.md CHANGELOG/${new_version}.md
-    elif [[ -e "CHANGELOG/revision.md" ]]; then
-        new_version="${old_version[0]}.${old_version[1]}.$((old_version[2] + 1))"
-        git mv CHANGELOG/revision.md CHANGELOG/${new_version}.md
+
+    if [[ -e "$breaking" ]]; then
+        if [[ "${old_version[0]}" == "0" ]]; then
+            new_version="0.$((old_version[1] + 1)).0"
+        else
+            new_version="$((old_version[0] + 1)).0.0"
+        fi
+        git mv "$breaking" CHANGELOG/${new_version}.md
+    elif [[ -e "$compatible" ]]; then
+        if [[ "${old_version[0]}" == "0" ]]; then
+            new_version="0.${old_version[1]}.$((old_version[2] + 1))"
+        else
+            new_version="${old_version[0]}.$((old_version[1] + 1)).0"
+        fi
```

Comment:
I’m fine with this approach, but I think we should describe somewhere that _our_ pre-1 semver handles feature additions as a 3rd-component increment rather than a 2nd-component one.